main/main.cpp

```diff
@@ -961,6 +961,7 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 
 	// Assigning here even though it's GLES2-specific, to be sure that it appears in docs
 	GLOBAL_DEF("rendering/quality/2d/gles2_use_nvidia_rect_flicker_workaround", false);
+	GLOBAL_DEF("rendering/quality/shading/gles2_high_float_precision.mobile", false);
```

Comment:
This should only be in VisualServer.cpp

Reply:
Indeed, you can add it after:
```
servers/visual_server.cpp
2446:   GLOBAL_DEF("rendering/gles2/compatibility/disable_half_float", false);
```
It could be renamed to `"rendering/gles2/compatibility/enable_high_float"` I guess to match the other `gles2/compatibility` setting?

Reply:
Actually this one should be removed now that it has been put in VisualServer. It should only be defined once.